apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -270,6 +275,15 @@ $(document).ready(() => {
 
   function onSchoolChange(_, event) {
     schoolData.ncesSchoolId = event ? event.value : '';
+    // ID is set to -1 and '' respectively as user toggles 'I cannot find my
+    // school above': exlude these from school selection events
+    if (!['-1', ''].includes(schoolData.ncesSchoolId)) {
+      analyticsReporter.sendEvent(
+        EVENTS.SCHOOL_SELECTED_FROM_LIST,
+        {ncesId: schoolData.ncesSchoolId},
+        PLATFORMS.BOTH
+      );
+    }
```

Comment:
Would it work to have an else case here that logs the `ADD_MANUALLY_CLICKED` instead of in `SchoolAutocompleteDropdownWithLabel.jsx`? (not sure if that's possible/makes more sense or not)

Reply:
I could see how that might make sense! However, I think the school ID getting updated is kind of a byproduct of clicking that checkbox, and I like having the event tied more directly to that checkbox in the codebase so someone reading through would be able to follow what the event is for. I think putting it here might be confusing